README.md

```diff
@@ -39,6 +39,19 @@ limitations under the License.
 
 ## Release Notes
 
+### v1.5.0
```

Comment:
Shouldn't we bump major verison? I think the changes we make are breaking (like the type of object returned from Parser etc)

Reply:
ok  updated